tools/pctools/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM registry.suse.com/bci/python:3.6
+
+ENV WKDIR=/home/venv
+
+# this dir will be a shared volume at runtime
+ENV SHARDIR=/home/tmp
+
+# script to install
+ARG VENV=./venv_inst.sh
+
+# venv dir and pip modules to install:
+# AWS
+ARG ENVAWS=aws 
+ARG PIPAWS="awscli"
+
+# Azure
+ARG ENVAZR=azure 
+ARG PIPAZR="azure-cli appdirs azure-identity importlib-metadata virtualenv"
+
+# EC2
+ARG ENVEC2=ec2 
+ARG PIPEC2="ec2imgutils"
+
+# Openstack
+ARG ENVOPS=opstk 
+ARG PIPOPS="openstacksdk python-openstackclient typing-extensions importlib-metadata importlib-resources colorama"
+
+# create the internal container's environment
+RUN cat /etc/os-release; mkdir $WKDIR; pip install virtualenv; echo Pre-virtualenv; python3 --version; pip --version
```

Comment:
I would concatenate the commands with `&&` otherwise we continue to run this also if there are failures.  
If here .eg. `pip install virtualenv` fails, the `RUN` will still succeed.

Simply replacing all `;` with `&&` should do the job